source/content/guides/drupal-9/drupal-9-hosted-createempty-md/15-deploy-live.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Migrate a Site That Was Created with an Empty Upstream to Drupal 9
+subtitle: Deploy Live
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-createempty-md/deploy-live
+anchorid: deploy-live
+editpath: drupal-9/drupal-9-hosted-createempty-md/15-deploy-live.md
+---
+
+<Partial file="drupal-9/deploy-using-relaunch.md" />
```

Comment:
Relaunch not needed. Just deploy from dev to staging and then prod

Reply:
updated